crawl-ref/source/ability.cc

```diff
@@ -3440,10 +3440,17 @@ static int _find_ability_slot(const ability_def &abil)
 
     // No requested slot, find new one and make it preferred.
 
-    // Skip over a-e (invocations), a-g for Elyvilon
+    // a-e is for invocations
     int first_slot = letter_to_index('f');
-    if (you_worship(GOD_ELYVILON))
-        first_slot = letter_to_index('h');
+
+    // Reserve the first ability slot (f) for Draconian breath
+    if (species_is_draconian(you.species)
+            && you.experience_level < 7)
```

Comment:
Maybe this should just check you.species == SP_BASE_DRACONIAN, so it wouldn't have to be modified if the XL when you grow up were changed?